### DISCUSSION TITLE
Adding some more DNS Provider

### DISCUSSION THREAD

**Discussion by macgrubi** [qdm12/ddns-updater#429] [Ideas]
Is there a possibility to add some other DNS provider for myself?
Or plan you add some more?

I am looking for 
**desec.io** 
[https://desec.readthedocs.io/en/latest/dyndns/configure.html#]
**two-dns**
[https://www.twodns.de/en/faqs]
**dnshome.de**
**cloudns.com**
**ipv64.net**

**Comment by dasIvan** [comment]
I would like to request the inclusion of "https://ipv64.net/" in your DDNS tool for DNS record updates.

Thank you for your consideration.
